website/docs/installation/skypilot/index.md

```diff
@@ -0,0 +1,83 @@
+# Skypilot Serving
+[SkyPilot](https://skypilot.readthedocs.io/en/latest/) is a framework for running LLMs, AI, and batch jobs on any cloud, offering maximum cost savings, highest GPU availability, and managed execution.
+
+[SkyServe](https://skypilot.readthedocs.io/en/latest/serving/sky-serve.html) is SkyPilot’s model serving library. SkyServe (short for SkyPilot Serving) takes an existing serving framework and deploys it across one or more regions or clouds.
```

Comment:
I'm wondering if it makes sense to mention the self-host / BYOC benefit somewhere. A sketch:

With this deployment mode, all replica VM(s) running Tabby and a lightweight controller VM are entirely deployed in your own cloud accounts and VPCs.

Reply:
Added